lib/openstudio/workflow/run.rb

```diff
@@ -298,11 +304,20 @@ def step
       def step_error(*args)
         # Make sure to set the instance variable @error to true in order to stop the :step
         # event from being fired.
-        @final_message = "Found error in state '#{@current_state}' with message #{args}}"
-        @logger.error @final_message
 
-        # transition to an error state
-        @current_state = :errored
+        # Allow continuing anyways if it fails in reporting measures
+        if @current_state == :reporting_measures
+          @final_message = "Found error in state '#{@current_state}' with message #{args}}"
+          @logger.error @final_message
+
+          next_state
+        else
+          @final_message = "Found error in state '#{@current_state}' with message #{args}}"
+          @logger.error @final_message
+
+          # transition to an error state
+          @current_state = :errored
+        end
```

Comment:
Here it is, if the state is :reporting_measure, still call `next_state` so the rest keeps going, including the cleanup to move html files to the reports/ directory